internal/nvmeof/controller/controllerserver.go

```diff
@@ -147,7 +148,9 @@ func (cs *Server) CreateVolume(
 	}()
 
 	// step 3: Populate volume context for NodeServer
-	populateVolumeContext(backend, nvmeofData)
+	if err := populateVolumeContext(backend, nvmeofData); err != nil {
```

Comment:
`CreateVolume()` should return a gRPC error, but `populateVolumeContext()` returns a "plain Go" error (as it should).